Rakefile

```diff
@@ -15,3 +15,9 @@ desc "Merge approved pull requests, without reviewing"
 task :merge_only do
   BulkMerger.merge_approved_pull_requests!
 end
+
+desc "List un-reviewed pull requests"
+task :list do
+  ENV["LIST"] = "true"
+  BulkMerger.approve_unreviewed_pull_requests!
```

Comment:
How about using an optional argument, rather than an environment variable? That would seem neater to me.

Reply:
Hmm. Done.